src/main/java/io/quarkus/gizmo2/impl/BlockCreatorImpl.java

```diff
@@ -902,7 +901,7 @@ private void doIf(final Expr cond, final Consumer<BlockCreator> whenTrue, final
         }
         doIfInsn(CD_void, cond, wt, wf);
     }
-
+    
```

Comment:
Obligatory formatting nitpicking :-)

Reply:
Uf, we should really configure some formatting tool...